message-index/messages/GHC-69158/example/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Name clash due to mix of qualified and unqualified re-export
+---
+
+## Error Message
+
+```
+Greeting.hs:3:5: error: [GHC-69158]
+    Conflicting exports for ‘greeting’:
+       ‘W.greeting’ exports ‘W.greeting’
+         imported qualified from ‘World’ at Greeting.hs:6:1-27
+         (and originally defined at World.hs:4:1-8)
+       ‘U.greeting’ exports ‘U.greeting’
+         imported qualified from ‘Universe’ at Greeting.hs:7:1-30
+         (and originally defined at Universe.hs:4:1-8)
+  |
+3 |   , U.greeting
+  |     ^^^^^^^^^^
+```
+
+## Explanation
+
+This examples defines the modules `World` and `Universe`, both of which export
+an identifier named `greeting`. The `Greeting` module then imports `World` and
+`Universe` qualified, meaning it can use both `greeting`s in the body of
```

Comment:
```suggestion
`Universe` qualified, meaning it can use both identifiers in the body of
```

Slightly too much use of "greeting"?

Reply:
I actually prefer the use of `greeting`s here, because it's more precise, if a bit repetetiev